libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/IndividualFeatureImportanceView/LargeIndividualFeatureImportanceView/LocalImportanceChart.tsx

```diff
@@ -0,0 +1,274 @@
+              <Stack.Item>
+                <ClassImportanceWeights
+                  onWeightChange={this.onWeightChange}
+                  selectedWeightVector={this.props.selectedWeightVector}
+                  weightOptions={this.props.weightOptions}
+                  weightLabels={this.props.weightLabels}
+                  disabled={this.props.isLocalExplanationsDataLoading}
+                />
+              </Stack.Item>
+            )}
+          </Stack>
+        </Stack.Item>
+      </Stack>
+    );
+  }
+
+  private setTopK = (newValue: number): void => {
+    this.setState({ topK: newValue });
+  };
+
+  private generateSortedData(): void {
+    const sortedData: Array<{ [key: string]: number[] | number | undefined }> =
+      [];
+    if (IsClassifier(this.props.modelType)) {
+      sortedData.push({
+        [this.props.weightOptions[0]]: this.getAbsoluteValues(
+          this.props.data?.precomputedExplanations?.localFeatureImportance
+            .scores[0][0]
+        )
+      });
+      sortedData.push(...this.addScores());
+    } else {
+      sortedData.push({
+        [regressionKeyValue]:
+          this.props.data?.precomputedExplanations?.localFeatureImportance
+            .scores[0]
+      });
+    }
+
+    this.setState({ sortedData });
+  }
+
+  private getAbsoluteValues(values?: number[]): number[] | undefined {
+    if (!values) {
+      return values;
+    }
+    const sortedScores = values.map((score) => Math.abs(score));
+    return sortedScores;
+  }
+
+  private addScores(): Array<{ [key: string]: number[] }> {
+    const scores: Array<{ [key: string]: number[] }> = new Array(
+      this.props.data?.precomputedExplanations?.localFeatureImportance.scores.length
+    );
+    this.props.data?.precomputedExplanations?.localFeatureImportance.scores.forEach(
+      (score, index) => {
+        scores.push({ [this.props.weightOptions[index + 1]]: score[0] });
+      }
+    );
+    return scores.filter((s) => s);
+  }
+
+  private getLocalImportanceBarOptions(): any {
+    const sortedData = getSortedData(
+      this.props.selectedWeightVector,
+      this.props.modelType,
+      this.state.sortedData,
+      this.state.sortAbsolute,
+      this.props.data?.precomputedExplanations.globalFeatureImportance
+        .feature_list,
+      this.props.rowNumber
+    );
+    const x = sortedData.map((d) => d.label);
+    const y = sortedData.map((d) => d.value);
+    let yAxisLabels: string = localization.Interpret.featureImportance;
+    if (this.props.modelType !== ModelTypes.Regression) {
+      const weightLabel =
+        this.props.weightLabels[this.props.selectedWeightVector];
+      yAxisLabels = yAxisLabels.concat(" - ", weightLabel);
+    }
+    const seriesData = [
+      {
+        color: getPrimaryChartColor(getTheme()),
+        data: y,
+        name: ""
+      }
+    ];
+    return {
+      chart: {
+        type: "column"
+      },
+      series: seriesData,
+      title: {
+        text: ""
+      },
```

Comment:
Is this intentional? If so, you probably don't need to specify it at all (?)

Reply:
Not intentional, I copied a format from other file and this came as is. Removed it.